lib/common.sh

```diff
@@ -946,8 +946,9 @@ common::set_cloud_binaries () {
     [[ -x $GSUTIL ]] && break
   done
 
-  # gcloud should be in the same place
-  GCLOUD=${GSUTIL/gsutil/gcloud}
+  for GCLOUD in "$GSUTIL/gsutil/gcloud" "$(which gcloud)"; do
+    [[ -x $GCLOUD ]] && break
```

Comment:
Can you make this an if statement instead of the && shorthand?

Reply:
Done.